larq_zoo/literature/__init__.py

```diff
@@ -0,0 +1,23 @@
+from larq_zoo.literature.binary_alex_net import BinaryAlexNet
+from larq_zoo.literature.birealnet import BiRealNet
+from larq_zoo.literature.densenet import (
+    BinaryDenseNet28,
+    BinaryDenseNet37,
+    BinaryDenseNet37Dilated,
+    BinaryDenseNet45,
+)
+from larq_zoo.literature.dorefanet import DoReFaNet
+from larq_zoo.literature.resnet_e import BinaryResNetE18
+from larq_zoo.literature.xnornet import XNORNet
+
+__all__ = [
```

Comment:
General question: does specifying `__all__` here actually do anything? It's literally a duplicated form of the code above and I *think* if you remove it you'll end up with the exact same functionality? At least e.g. `XNORNet` will be exposed this way. Though I'm not sure if specifying `__all__` here prevents you from manually accessing e.g. `larq_zoo.literature.xnornet.TrainXNORNet`?

Reply:
For your last point: you can't stop people from manually accessing anything in Python really.

Reply:
`__all__` controls what happens if you do `from larq_zoo import *`. I think editors/IDEs might also use it.

Reply:
> `__all__` controls what happens if you do `from larq_zoo import *`. I think editors/IDEs might also use it.

I see, thanks for the clarification. Indeed if I print `globals()` without specifying `__all__`, a lot more shows up. A shame that Python doesn't automatically recognize this as limiting imports...